bin/study.js

```diff
@@ -1,5 +1,25 @@
+'use strict';
+
 // Repeat Ruby Lab for JS
-// let myArray = [1, 2, 3, 4, 5];
+let myArray = [1, 2, 3, 4, 5];
+
+// -   Assign `20` to the element at `length+1`.
+myArray[5] = 20;
+
+// -   Access the 3rd element from the end of an array length of 5.
+myArray = [1, 2, 3, 4, 5];
+myArray[2];
```

Comment:
This works perfectly because we know how long our array is. But if we didn't, we could something like `myArray.legnth-3`